.buildscript/deploy_snapshot.sh

```diff
@@ -6,7 +6,7 @@
 # http://benlimmer.com/2013/12/26/automatically-publish-javadoc-to-gh-pages-with-travis-ci/
 
 SLUG="square/javapoet"
-JDK="oraclejdk8"
+JDK="oraclejdk7"
```

Comment:
... a closer reading tells me that if we can successfully build on JDK7, then a JDK8-built binary targeting JDK7 should work fine. Your call.